REST Bindings/openapi.yaml

```diff
@@ -1016,22 +882,42 @@ paths:
         - $ref: '#/components/parameters/GS1-CBV-XML-Format'
         - $ref: '#/components/parameters/GS1-EPCIS-Extensions'
         - $ref: '#/components/parameters/GS1-CBV-Extensions'
+
+
       description: |
-        This endpoint accesses individual events with the electronic product code.
+        This endpoint helps to navigate EPCIS events by electronic product codes. It returns
+        EPCIS events up to the amount defined in `perPage`. The server returns a `Link` header to point to the remaining
+        results. Optionally, EPCIS events can be further filtered using the EPCIS Query Language in as query string parameters.
+
+        Example 1 - EPC in a URN form with EPCIS Query Language filtering
+
+        ```
+        https://example.com/epcs/urn:epc:id:sgtin:0614141.107346.2018?EQ_bizStep=urn%3Aepcglobal%3Acbv%3Abizstep%3Ashipping%2Curn%3Aepcglobal%3Acbv%3Abizstep%3Adecommissioning&GE_eventTime=2015-03-15T00%3A00%3A00.000-04%3A00
```

Comment:
```suggestion
        https://example.com/epcs/urn:epc:id:sgtin:0614141.107346.2018?EQ_bizStep=shipping%2Cdecommissioning&GE_eventTime=2015-03-15T00%3A00%3A00.000-04%3A00
```

Reply:
Mark  @mgh128 to review e-mails of 27-28 October regarding potential use of non-escaped vertical pipe as logical OR separator - double-check that this can work.  If so, replace %2C in this example at line 895 and elsewhere.